pages/collections/_type/index.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <b-container>
+    <ContentHeader
+      :title="$tc('entity.index.' + this.$route.params.type, 2)"
+    />
+    <b-row class="flex-md-row pb-5">
+      <b-col cols="12">
+        <b-card-group
+          class="card-deck-4-cols"
+          deck
+          :data-qa="`${ this.$route.params.type} listing page`"
+        >
+          <ContentCard
+            v-for="entity in entities"
+            :key="entity.id"
+            :title="entity.prefLabel[$i18n.locale]"
```

Comment:
This needs to be using our lang map lookup logic with fallback to English, i.e. `langMapValueForLocale` from plugins/europeana/utils.js

Otherwise some will have no title.

Examples, for English UI:
* Concept 572 on page 3 should have title "Progressiu" from `ca` lang
* Concept 702 on page 4 should have title "Alternative" from `de` lang

Reply:
The ContentCard component already uses langMapValueForLocale for displayTitle so isn't this logic already implemented?

Reply:
Yes, so change this line to:
```
            :title="entity.prefLabel"
```
Then the examples will have the fallback languages used.

Reply:
ok I see, thanks